test/suites/standard/log-scan.robot

```diff
@@ -0,0 +1,51 @@
+*** Settings ***
+Documentation       Tests for various log messages we do or do not want.
+
+Resource            ../../resources/common.resource
+Resource            ../../resources/microshift-process.resource
+Library             ../../resources/journalctl.py
+
+Suite Setup         Setup
+Suite Teardown      Teardown
+
+Test Tags           restart    slow
+
+
+*** Variables ***
+${CURSOR}       ${EMPTY}    # The journal cursor before restarting MicroShift
+
+
+*** Test Cases ***
+Should Not Find Forbidden
+    [Documentation]    Logs should not say "forbidden"
+    Pattern Should Not Appear In Log Output    ${CURSOR}    forbidden
+
+Should Not Find Cannot Patch Resource
+    [Documentation]    Logs should not say "cannot patch resource"
+    Pattern Should Not Appear In Log Output    ${CURSOR}    cannot patch resource
+
+Should Find Etcd Is Ready
+    [Documentation]    Logs should say "etcd is ready"
+    Pattern Should Appear In Log Output    ${CURSOR}    etcd is ready
+
+Should Find MicroShift Is Ready
+    [Documentation]    Logs should say "microshift is ready"
+    Pattern Should Appear In Log Output    ${CURSOR}    microshift is ready
```

Comment:
Do we want to run these at the end of the other suites? Asking because I think it would be interesting as some sort of pre-suite-teardown instead of standalone tests. You would catch a wider variety of logs, as right now the test suite execution order is randomized.

Reply:
Today I'm mostly interested in looking for errors just from starting the service itself, not things triggered by the tests. I expect any test that might trigger an error to do its own check (the keywords here are all reusable for that).

Reply:
What about executing this one right after https://github.com/openshift/microshift/blob/main/test/bin/scenario.sh#L454-L461 ? That way you always ensure you are catching both the first start and the whole test. A test doing restarts might escape these checks because of randomization.

Reply:
I'm trying to avoid the global invariant pattern. This suite checks that we don't generate a few specific errors unrelated to any tests. After we merge it, we will have the keywords in place to allow each test suite to check conditions relevant to it, so that if we start seeing failures we know why we're checking those messages.